src/pathfora.js

```diff
@@ -572,6 +573,16 @@
         core.initializeScrollWatchers(widget);
       }
 
+      if (condition.customTrigger) {
```

Comment:
kind of a nit but i'd call this manualTrigger rather than custom probably. that was its clear that it relies on manual client side js.